### ISSUE TITLE
Meetbot: make link re-posting optional

### ISSUE THREAD

**Issue by MoralCode** [fedora-infra/matrix-bots#21]
I've encountered the following situation a couple times in the past week or so during online matrix only meetings in the fedora community:

1. someone posts a link in the meeting as part of the conversation
2. the meeting organizer/point person issuing the bot commands issues `!link` to add the link to the notes
3. the bot replies with the link (presumably as confirmation that it worked)

example from todays community ops meeting:

![Screenshot from 2024-05-28 10-13-06](https://github.com/fedora-infra/matrix-bots/assets/17362949/75ee18c4-caa1-4790-9ef1-98ad4faee888)
![Screenshot from 2024-05-28 10-52-12](https://github.com/fedora-infra/matrix-bots/assets/17362949/61cd29ab-cd03-48e2-97ec-84555672313b)

This seems like it may be a lot of over-duplication of links in meeting rooms. is this likely something that could be easily scoped for a good first issue?

**Comment by ryanlerch** [comment]
Those responses are relatively new addtions to the meetbot bot, and were added as part of the much requested multiple commands in a single multiple line message support.

Previously, meetbot just put an emoji response on a message, but didnt respond -- now becuase we are in interpreting multiple commands out of a single message, we cannot do that anymore (only a single emoji per tag).

I feel we need some sort of response from meetbot that they have logged the tag properly.

Also, it might be worth posting this in the upstream for the meetbot plugin itself. (i cant transfer it becuase its not in the fedora-infra org) https://github.com/GregSutcliffe/maubot-meetings

**Comment by MoralCode** [comment]
> I feel we need some sort of response from meetbot that they have logged the tag properly.

Yeah I agree - would it be possible for the bot to switch between an emoji react and a new message depending on if theres one or more than one link detected in a post? Seems like one link is still by far the dominant usecase

**Comment by MoralCode** [comment]
> Also, it might be worth posting this in the upstream for the meetbot plugin itself. (i cant transfer it becuase its not in the fedora-infra org) https://github.com/GregSutcliffe/maubot-meetings

Would it be helpful if it was transferred? or has that been tried before?
